src/views.njk

```diff
@@ -1,23 +1,25 @@
 ---
+title: Views
 permalink: /views/{% if pagination.pageNumber > 0 %}page/{{ pagination.pageNumber + 1 }}/{% endif %}
 pagination:
   data: collections.views
   size: 10
 ---
 {% extends 'layouts/base.njk' %}
 
-{% set currentPage = pagination.pageNumber + 1 %}
-{% set pageTitle = 'Views (Page ' + currentPage + ')' %}
-
 {% block content %}
   <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
     <symbol id="placeholder" viewBox="-4 -4 224 224">
       {% include 'svg/placeholder.svg' %}
     </symbol>
   </svg>
-  
-  <article class="views">
+
+  <article class="views static-view">
 		<h1>Views</h1>
+    {% set searchAction = "/views/" %}
+    {% set placeholderForSearch = "Search posts ..." %}
```

Comment:
In `header-nav.njk` line 11 there is no space before the ellipsis ('…') for the `placeholder` value. Here there is. It's probably best to make these consistent. Either both have space before … or neither has a space before …

Reply:
The formats are synced up by removing the space between "search posts" and the following ellipsis. Thanks for the catch.